lib/src/Pos/Launcher/Runner.hs

```diff
@@ -97,201 +56,107 @@ cslMain withLayers control = withLayers $ \(diffusionLayer, logicLayer) ->
 runRealMode
     :: forall ext ctx a.
        (HasCompileInfo, WorkMode ctx (RealMode ext))
-    => NodeResources ext (RealMode ext)
+    => NodeResources ext
     -> (ActionSpec (RealMode ext) a, OutSpecs)
     -> Production a
-runRealMode = runRealBasedMode @ext @ctx identity identity
+runRealMode = runRealBasedMode @ext
```

Comment:
Since `runRealMode = runRealBasedMode`, I think `runRealBasedMode` can be removed.

Reply:
Will do